Mathlib/Analysis/Convex/Cone/Pointed.lean

```diff
@@ -0,0 +1,260 @@
+/-
+Copyright (c) 2022 Apurva Nakade All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Apurva Nakade
+-/
+import Mathlib.Analysis.Convex.Cone.Dual
+import Mathlib.Algebra.Order.Nonneg.Ring
+import Mathlib.Algebra.Module.Submodule.Basic
+
+
+/-!
+# Pointed cones
+
+We define a pointed cones as convex cones which contain `0`. This is a bundled version of
+`ConvexCone.Pointed`. Pointed cones have a nicer algebraic structure than convex cones. They form
+a submodule of the ambient space when the scalars are restricted to being positive. This allows us
```

Comment:
I misread this originally: I thought you meant there is an instance `Module {x : k // 0 <= x} (PointedCone k E)`. Can you clarify the module doc?

Reply:
This is indeed what it means! The instance comes from being a `Submodule` although it is never explicitly defined.

Reply:
Hopefully this will get much clearer once I redefine `PointedCone`.